src/devtools/client/inspector/markup/new-markup.js

```diff
@@ -168,8 +183,7 @@ class MarkupView {
    */
   async expandNode(nodeId) {
     await this.getChildren(this.nodes.get(nodeId));
-    this.tree[nodeId].isExpanded = true;
-    this.store.dispatch(updateTree(this.tree));
+    this.store?.dispatch(updateNodeExpanded(nodeId, true));
```

Comment:
when would store not be defined?

Reply:
If `destroy()` was called during the `await` in the previous line.